ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/clientlibs/editor/js/panelcontainer.js

```diff
@@ -40,10 +40,11 @@
             } else if (dialogContent.querySelector("[data-cmp-container-v1-dialog-policy-hook]")) {
                 handlePolicyDialog(dialogContent);
             }
-
-            handleWrapData(dialogContent);
         }
-
+        // For handling the case when tabs,accordion and wizard inherit panel edit-dialog
+        if($dialog[0]) {
```

Comment:
not clear. earlier dialogContent was being passed but now a different arg is passed.

Reply:
`dialogContent` uses the selector `dialogContent: ".cmp-adaptiveform-panelcontainer__editdialog"` which won't work in case of tabs, accordion and wizard.
where as `$dialog` is generic.